packages/code.rb

```diff
@@ -0,0 +1,57 @@
+require 'package'
+
+class Code < Package
+  description 'Visual Studio Code is a source code editor developed by Microsoft for Windows, Linux and macOS.'
+  homepage 'https://code.visualstudio.com/docs/?dv=linux64'
+  version '1.25'
+  case ARCH
+  when "x86_64"
+    source_url 'https://go.microsoft.com/fwlink/?LinkID=620884'
+    source_sha256 '5856bbebf38aa05d584da4722869bbe507cf123f69f7ffab5f1532d73dbf3438'
+  when "i686"
+    source_url 'https://go.microsoft.com/fwlink/?LinkID=620885'
+    source_sha256 'af6adc2e2500e50bfebe7ee7b97d661b6e774a590136bf5f89334132a5b292e2'
+  else
+    source_url 'https://raw.githubusercontent.com/Microsoft/vscode/1.25.1/README.md'
+    source_sha256 'c1c5e6ec903730a4e116c1cfd83bb29acd227195d97f2ec8944452808232c310'
+  end
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'sommelier'
+  depends_on 'xterm'
+  depends_on 'libgconf'
+  depends_on 'xdg_base'
+
+  def self.install
+    case ARCH
+    when 'armv7l', 'aarch64'
```

Comment:
It would be better to have conditional logic that identifies the architecture that is supported rather than unsupported.  So something like:
````
case ARCH
when 'i686', 'x86_64'
  ...
else
  puts
  puts "Visual Studio Code does not currently support your architecture.".lightred
  ...
end
```